themes/default/content/blog/redmonk-pulumi-developer-first-infra/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Pulumi and RedMonk discuss developer-first infrastructure and why it matters"
+date: 2022-04-13T17:46:27-05:00
+meta_desc: |
```

Comment:
update to: 

Pulumi and RedMonk discuss how developer-first infrastructure means empowering developers to build and deploy applications through software engineering practices in cloud computing.